en/heroku/README.md

```diff
@@ -98,6 +98,12 @@ DATABASES = {
 
 db_from_env = dj_database_url.config(conn_max_age=500)
 DATABASES['default'].update(db_from_env)
+
+# at the very end if the file add:
```

Comment:
You probably mean <pre># at the very end <strong>of</strong> the file, add:</pre>